Assets/Scripts/Map/PlatformAttach.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+public class PlatformAttach : MonoBehaviour
+{
```

Comment:
There should be no line break before the opening curly brace. E.g. `MonoBehaviour {`